### PR TITLE
Fix release Linux packages didn't auto-upload to the release log

### PR DESCRIPTION
**Pull request by karelrooted**
Fix failed to release linux packages (directly release instead of rely on upload-artifact and then download-artifact)

### CODE REVIEW

**Comment by git-hulk** [comment]
Really cool!!!

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -100,26 +103,13 @@ jobs:
...
         uses: softprops/action-gh-release@v1
         with:
           files: |
-            ./*.deb
             ./*.rpm
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+
+
```
Comment (ShooterIT) [inline .github/workflows/release.yaml]: two extra empty lines

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -100,26 +103,10 @@ jobs:
...
-        uses: actions/download-artifact@v2
-
       - name: Release
         uses: softprops/action-gh-release@v1
         with:
           files: |
-            ./*.deb
             ./*.rpm
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (ShooterIT) [inline .github/workflows/release.yaml:112]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```
we should keep one empty line 🤣

**Review by ShooterIT — APPROVED** [review]

File: .github/workflows/release.yaml (hunk 2 above)
Comment (karelrooted) [reply .github/workflows/release.yaml:112]: kvrocks.yaml did't have one empty line, change kvrocks.yaml to have one empty line too in this PR？

Comment (ShooterIT) [reply .github/workflows/release.yaml:112]: oh, my bad, i forget, it doesn't matter, to make this PR doesn't influence other things, we needn't

**Review by git-hulk — APPROVED** [review]

**Pull request merged**